ledger/api-server-damlonx/reference-v2/src/test/scala/com/daml/ledger/api/server/damlonx/reference/v2/EphemeralPostgresReferenceServerMain.scala

```diff
@@ -10,3 +10,7 @@ object EphemeralPostgresReferenceServerMain extends App with PostgresAround {
   sys.addShutdownHook(stopAndCleanUp(fixture.tempDir, fixture.dataDir, fixture.logFile))
   ReferenceServer.main(args ++ List("--jdbc-url", fixture.jdbcUrl))
 }
+
+object PlainReferenceServerMain extends App {
```

Comment:
`App` uses `DelayedInit` right now, which can end up causing some weird behavior. I would suggest using a simple `main` method (if we really want to keep this at all, as per my previous comment).

Reply:
Removing